ci-operator/config/openshift/runtime-utils/openshift-runtime-utils-master.yaml

```diff
@@ -0,0 +1,40 @@
+base_images:
+  os:
+    cluster: https://api.ci.openshift.org
+    name: centos
+    namespace: openshift
+    tag: "7"
+binary_build_commands: make build
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
+    name: release
+    namespace: openshift
+    tag: golang-1.12
+promotion:
+  name: "4.3"
+  namespace: ocp
+resources:
+  '*':
+    limits:
+      memory: 12Gi
+    requests:
+      cpu: "3"
+      memory: 8Gi
+tag_specification:
+  name: "4.3"
+  namespace: ocp
+test_binary_build_commands: make test
```

Comment:
the point of this field as i understand it is to provide a command to build a "test binary" (e.g. openshift.test in origin) that you can then invoke from the "test-bin" image.  Your repo doesn't produce a test binary, so you do not need this.

Reply:
removed